CodeConverter/Util/SyntaxTokenExtensions.cs

```diff
@@ -146,8 +146,12 @@ public static SyntaxToken WithoutSourceMapping(this SyntaxToken token)
 
         public static SyntaxTokenList RemoveOnly(this SyntaxTokenList list, Func<SyntaxToken, bool> where)
         {
-            var toRemove = list.OnlyOrDefault(where);
-            if (toRemove != default) list = list.Remove(toRemove);
+            for (int i = 0; i < list.Count; ++i) {
+                if (!where(list[i])) continue;
+
+                list = list.RemoveAt(i--);
+            }
+
             return list;
```

Comment:
Would probably rename this to `RemoveWhere` - "only" was supposed to imply only one, though it's now obvious that meaning is ambiguous in this case anyway